FirebaseRemoteConfig/Tests/SwiftAPI/Resources/README.md

```diff
@@ -0,0 +1,66 @@
+# Remote Config Console API
+
+[`RemoteConfigConsole.swift`](https://github.com/firebase/firebase-ios-sdk/blob/nc-rc-test-api/FirebaseRemoteConfig/Tests/SwiftAPI/RemoteConfigConsole.swift) provides a simple API for interacting with an app's remote config on the Firebase console.
+
+## Local Development Setup
+
+You can start by generating the `FirebaseRemoteConfig` project:
+```bash
+pod gen FirebaseRemoteConfig.podspec --local-sources=./ --auto-open --platforms=ios
+```
+
+Next, create an accompanying Firebase project on the console and download the  `GoogleService-Info.plist`. Then, drag the `GoogleService-Info.plist` into the generated Remote Config Xcode project.
+
+While the `RemoteConfigConsole` API basically just makes simple network calls, we will need to include an `access token` so our requests do the proper "handshake" with the Firebase console.
+
+To generate the access token, we will need a **Firebase Service Account Private Key**
+
+### Generating the Firebase Service Account Private Key
+This private key is needed to create an access token with the valid parameters that authorizes our requests to programmtically make changes to remote config on the Firebase console.
+
+Go to the Firebase console and navigate to your project's settings. Click on the **Service accounts** tab and and then generate the private key by clicking the blue button that says "Generate new private key"
+
+A `.json` file will be downloaded. Go ahead and rename this file to `ServiceAccount.json` and move it to your `$HOME/.credentials/` directory. You may have to create the `.credentials/` directory.
+
+### Create the Access Token
+We use Google's [Auth Library for Swift](https://github.com/googleapis/google-auth-library-swift) to generate the access token. There are a few example use cases provided. We will use the [`TokenSource`](https://github.com/googleapis/google-auth-library-swift/blob/master/Sources/Examples/TokenSource/main.swift) example.
+
+As you can see below when we set the `GOOGLE_APPLICATION_CREDENTIALS` env variable. This library will use the `ServiceAccount.json` file we generated earlier to create our access token.
+
+#### Terminal Commands
+```bash
+$ git clone git@github.com:googleapis/google-auth-library-swift.git
```

Comment:
Can this be a script?

Reply:
Yes, I can add one that does just that for local development. Just wanna add, we can't use the one we use in GitHub actions without knowing the secrets password. Its designed as of now to be used in a single workflow step so we had an easy way to generate tokens for a project in the future (if we add more Admin enabled testing APIs for other Firebase products).

So for local remote config development, the only catch is that you'll need to keep a copy of the unencrypted service account file for the internal rc project (or know the GitHub secrets password to run the script i wrote locally). If someone in the community wants to contribute, they can follow the steps in the readme to creating a demo project and downloading that project's affiliated service account file and so on.

Reply:
See the `generate_access_token.sh` script in this PR. I changed it up to be able to run both in GHA and for local development!